data/tabular/chemdner/transform.py

```diff
@@ -0,0 +1,27 @@
+from datasets import load_dataset
+
+from chemnlp.data.utils import oxford_comma_join
+
+
+def process():
+    dataset = load_dataset("kjappelbaum/chemnlp-chemdner")
+    df = dataset["train"].to_pandas()
+
+    matched_words = []
+    for ent in df["entities"]:
+        if len(ent) == 0:
+            matched_words.append("no match")
+        else:
+            matched_words.append(oxford_comma_join(ent))
+
+    df["matched_words"] = matched_words
+    df["sentence"] = df["text"]
```

Comment:
Apply sentence cleaner here too?

Reply:
I guess we only need it for the data from Santosh, which is why you don't apply it here?

Reply:
the sentences in this dataset have been cleaned before uploading to HF